src/Umbraco.Web.UI/Umbraco.Web.UI.csproj

```diff
@@ -1973,6 +1973,8 @@
     <Content Include="Views\Partials\Grid\Editors\Textstring.cshtml" />
     <Content Include="Views\Partials\Grid\Bootstrap2.cshtml" />
     <Content Include="Views\Partials\Grid\Editors\Base.cshtml" />
+    <Content Include="Views\Partials\Grid\Bootstrap3-Fluid.cshtml" />

```

Comment:
... well no they're not in the release package since they aren't in the csproj... but we have them in the source, added at the same time as the others:
https://github.com/umbraco/Umbraco-CMS/tree/dev-v7/src/Umbraco.Web.UI/Views/Partials/Grid